### ISSUE TITLE
Error in train.cytonorm

### ISSUE THREAD

**Issue by mayanbriller** [ImmuneDynamics/Spectre#34]
Hello!
While running the workflow for batch correction, I am getting the following:

```
Training alignment - file (batch) preparation
Training alignment - file (batch) and metacluster splitting
 -- running File '1'
Error in flowCore::write.FCS(file.ff[cellClusterIDs == cluster, ], file = file.path(getwd(),  : 
  Argument 'filename' has to be a character of length 1.
Calls: train.cytonorm
Execution halted
```

I noticed that no where in the workflow do we define `fine.dir` , so I defined it as 
```
setwd("Output 2.4 - fine alignment")
fine.dir = getwd()
```

Could it be the reason for this error?
Thanks
Mayan

**Comment by tomashhurst** [comment]
Hi @mayanbriller ,

Could I ask what OS you are using? The filename issue is a little weird. Does it run with the demo dataset?

Tom

**Comment by mayanbriller** [comment]
I am working on Google Cloud. 
I was able to reproduce this error with the demo data.

I don't think it's related - but for the purposes of testing the workflow I deleted the parts of the code that produce graphs. Do you think it could be related?

```
Training alignment - file (batch) preparation
Training alignment - file (batch) and metacluster splitting
 -- running File '1'
Error in flowCore::write.FCS(file.ff[cellClusterIDs == cluster, ], file = file.path(getwd(),  : 
  Argument 'filename' has to be a character of length 1.
Calls: train.cytonorm
Execution halted
sh: 0: getcwd() failed: No such file or directory
```


> 
> Hi @mayanbriller ,
> 
> Could I ask what OS you are using? The filename issue is a little weird. Does it run with the demo dataset?
> 
> Tom

**Comment by tomashhurst** [comment]
Should be fixed in [v1.2.0](https://github.com/ImmuneDynamics/Spectre/releases/tag/v1.2.0)
